pkg/configs/configs.go

```diff
@@ -182,6 +182,23 @@ func (c RulesConfig) Parse() (map[string][]rules.Rule, error) {
 	}
 }
 
+// ParseFormatted returns the rulefmt map of a users rules configs. It allows
+// for rules to be mapped to disk and read by the prometheus rules manager.
+func (c RulesConfig) ParseFormatted() (map[string]rulefmt.RuleGroups, error) {
```

Comment:
Does this deal with rules format "v1" (as the old `Parse()` function did) ?

Reply:
I added V1 support with tests cases.